pkg/cmd/pkg/package_uninstall.go

```diff
@@ -24,3 +99,63 @@ func newCmdPackageUninstall(ctx api.Context) *cobra.Command {
 	cmd.Flags().BoolVar(&yes, "yes", false, "Disable interactive mode and assume “yes” is the answer to all prompts")
 	return cmd
 }
+
+func removeCLI(ctx api.Context, packageName string) error {
+	cluster, err := ctx.Cluster()
+	if err != nil {
+		return err
+	}
+
+	manager := ctx.PluginManager(cluster)
+	return manager.Remove(packageName)
+}
+
+func packageInstallCount(c *cosmos.Client, packageName string) (int, error) {
+	packages, err := c.PackageList()
+	if err != nil {
+		return 0, err
+	}
+
+	for _, p := range *packages {
```

Comment:
Not sure why PackageList returns a pointer to a slice, this should just be a slice.